shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-opengauss/src/main/java/org/apache/shardingsphere/scaling/opengauss/component/OpenGaussWalDumper.java

```diff
@@ -123,6 +131,21 @@ private void dump() {
         }
     }
 
+    private void updateRecordOldValue(final Record record) {
+        if (!(record instanceof DataRecord)) {
+            return;
+        }
+        DataRecord dataRecord = (DataRecord) record;
+        if (!ScalingConstant.UPDATE.equals(dataRecord.getType())) {
+            return;
+        }
+        for (Column col: dataRecord.getColumns()) {
+            if (col.isPrimaryKey() && col.isUpdated()) {
+                col.setOldValue(col.getValue());
+            }
```

Comment:
Could we init `oldValue` with `Column`'s full fields constructor, just like `DataRecordMerger.mergeColumn(DataRecord, DataRecord)`?

Reply:
The Column constructor was called in WalEventConverter::convert,  modify will influence pg. so I moved here ,and only effect on openGuass.

Reply:
Seems PostgreSQL and openGauss could not get `beforeRows` from `UpdateRowsEvent`, not like MySQL binlog. And update event include all columns.
```
scaling1=# update t_order_0 set status='ok1' where order_id=1;
UPDATE 1
scaling1=# SELECT * FROM pg_logical_slot_get_changes('regression_slot', NULL, NULL);
    lsn    | xid |                                                  data
-----------+-----+--------------------------------------------------------------------------------------------------------
 0/17B18A8 | 599 | BEGIN 599
 0/17B18A8 | 599 | table public.t_order_0: UPDATE: order_id[integer]:1 user_id[integer]:2 status[character varying]:'ok1'
 0/17B1CB8 | 599 | COMMIT 599
(3 rows)
```

We'll make `Column.oldValue` not final for now.